FinancialTransactionsApi/V1/Helpers/FileGenerator.cs

```diff
@@ -0,0 +1,131 @@
+using CsvHelper;
+using CsvHelper.Configuration;
+using FinancialTransactionsApi.V1.Boundary.Response;
+using FinancialTransactionsApi.V1.Domain;
+using NodaMoney;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using WkHtmlToPdfDotNet;
+using WkHtmlToPdfDotNet.Contracts;
+
+namespace FinancialTransactionsApi.V1.Helpers
+{
+    public static class FileGenerator
+    {
+        public static byte[] WritePdfFile(List<Transaction> transactions, string name, string period, IConverter converter)
```

Comment:
Could you please provide me with the link to the api specs for this functionality?

Reply:
Will do that